### ISSUE TITLE
problem 

### ISSUE THREAD

**Issue by MehmetAliCan10** [Pure-L0G1C/Instagram#29]
Traceback (most recent call last):
  File "instagram.py", line 8, in <module>
    from lib.bruter import bruter 
  File "/home/ubuntu/Downloads/Instagram-master/lib/bruter.py", line 5, in <module>
    from .queue import queue 
  File "/home/ubuntu/Downloads/Instagram-master/lib/queue.py", line 5, in <module>
    from requests import get
ImportError: No module named requests

**Comment by Pure-L0G1C** [comment]
You have to install the requirements

**Comment by MehmetAliCan10** [comment]
ubuntu@ubuntu:~$ python '/home/ubuntu/Desktop/Instagram-master/instagram.py' 
Traceback (most recent call last):
  File "/home/ubuntu/Desktop/Instagram-master/instagram.py", line 8, in <module>
    from lib.bruter import Bruter 
  File "/home/ubuntu/Desktop/Instagram-master/lib/bruter.py", line 5, in <module>
    from .queue import Queue 
  File "/home/ubuntu/Desktop/Instagram-master/lib/queue.py", line 5, in <module>
    from requests import get
ImportError: No module named requests

Also work not have newest ubuntu and python updated also work not :/

**Comment by Pure-L0G1C** [comment]
You need to install the requirements

**Comment by MehmetAliCan10** [comment]
What is requirements?

**Comment by MehmetAliCan10** [comment]
What is command for requirements to install?

**Comment by Pure-L0G1C** [comment]
I just updated the README file, it should show you how.

**Comment by MehmetAliCan10** [comment]
Ok But can you maybe private picture screen see without hacking password ?


Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________
Van: Mohamed <notifications@github.com>
Verzonden: Tuesday, May 15, 2018 6:53:36 PM
Aan: Pure-L0G1C/Instagram
CC: MehmetAliCan10; Author
Onderwerp: Re: [Pure-L0G1C/Instagram] problem (#29)

I just updated the README file, it should show you how.

**Comment by MehmetAliCan10** [comment]
~$ sudo pip install -r requirements.txt
Traceback (most recent call last):
  File "/usr/bin/pip", line 9, in <module>
    from pip import main
ImportError: cannot import name main

Have again problem fix it ...

**Comment by WhoseSomebody** [comment]
@MehmetAliCan10 
If it's still open for you. I faced the same issue.
The reason was that I had both python 2.7 & 3.6 installed. And default command to install dependencies didn't work for me.
To install dependencies from `requirements.txt` I had to do `[sudo] pip3 install ...` for each of required libs (bs4, future, requests, colorama). 
`
$  pip3 install requests
$  pip3 install future
$  pip3 install colorama
$  pip3 install bs4
`

And it fortunately helped me. My OS is MacOS Mojave. Maybe that was the case. Good luck!
